dev/cognitiveservices/data-plane/Language/analyzetext.json

```diff
@@ -1681,62 +2510,865 @@
-        "piiCategories": {
-          "$ref": "#/definitions/PiiCategories"
-        },
-        "stringIndexType": {
-          "$ref": "common.json#/definitions/StringIndexType"
-        }
+    "VolumeUnit": {
+      "type": "string",
+      "enum": [
+        "Unspecified",
+        "CubicMeter",
+        "CubicCentimeter",
+        "CubicMillimeter",
+        "Hectoliter",
+        "Decaliter",
+        "Liter",
+        "Centiliter",
+        "Milliliter",
+        "CubicYard",
+        "CubicInch",
+        "CubicFoot",
+        "CubicMile",
+        "FluidOunce",
+        "Teaspoon",
+        "Tablespoon",
+        "Pint",
+        "Quart",
+        "Cup",
+        "Gill",
+        "Pinch",
+        "FluidDram",
+        "Barrel",
+        "Minim",
+        "Cord",
+        "Peck",
+        "Bushel",
+        "Hogshead"
+      ],
+      "x-ms-enum": {
+        "name": "VolumeUnit",
+        "modelAsString": true
       },
-      "allOf": [
-        {
-          "$ref": "#/definitions/PreBuiltTaskParameters"
-        }
-      ]
+      "description": "The Volume Unit of measurement"
     },
-    "PiiLROTask": {
-      "type": "object",
-      "description": "An object representing the task definition for a PII Entities Recognition task.",
-      "properties": {
-        "parameters": {
-          "$ref": "#/definitions/PiiTaskParameters"
+    "SpeedUnit": {
+      "type": "string",
+      "enum": [
+        "Unspecified",
+        "MetersPerSecond",
+        "KilometersPerHour",
+        "KilometersPerMinute",
+        "KilometersPerSecond",
+        "MilesPerHour",
+        "Knots",
+        "FeetPerSecond",
+        "FeetPerMinute",
+        "YardsPerMinute",
+        "YardsPerSecond",
+        "MetersPerMillisecond",
+        "CentimetersPerMillisecond",
+        "KilometersPerMillisecond"
+      ],
+      "x-ms-enum": {
+        "name": "SpeedUnit",
+        "modelAsString": true
+      },
+      "description": "The speed Unit of measurement"
+    },
+    "AreaUnit": {
+      "type": "string",
+      "enum": [
+        "Unspecified",
+        "SquareKilometer",
+        "SquareHectometer",
+        "SquareDecameter",
+        "SquareDecimeter",
+        "SquareMeter",
+        "SquareCentimeter",
+        "SquareMillimeter",
+        "SquareInch",
+        "SquareFoot",
+        "SquareMile",
+        "SquareYard",
+        "Acre"
+      ],
+      "x-ms-enum": {
+        "name": "AreaUnit",
+        "modelAsString": true
+      },
+      "description": "The area Unit of measurement"
+    },
+    "LengthUnit": {
+      "type": "string",
+      "enum": [
+        "Unspecified",
+        "Kilometer",
+        "Hectometer",
+        "Decameter",
+        "Meter",
+        "Decimeter",
+        "Centimeter",
+        "Millimeter",
+        "Micrometer",
+        "Nanometer",
+        "Picometer",
+        "Mile",
+        "Yard",
+        "Inch",
+        "Foot",
+        "LightYear",
+        "Point"
+      ],
+      "x-ms-enum": {
```

Comment:
By GA, everything should be documented.

Reply:
@bidisha-c can you create a work item to track this?